components/frontend/src/header_footer/Menubar.js

```diff
@@ -99,7 +100,10 @@ function Logout({ user, email, set_user }) {
                     text: "Logout",
                     icon: "log out",
                     onClick: () => {
-                        logout().then(() => set_user(null))
+                        logout()
+                            .then(() => set_user(null))
+                            .catch((error) => console.log(error))
+                            .finally(() => set_user(null))
```

Comment:
Why not always `set_user(null)` after logout and only catch the error here?

Reply:
Yeah, good point.